doc/releases/changelog-0.40.0.md

```diff
@@ -136,22 +136,68 @@ qubit operators.
   Additional fine-tuning is available within each function, such as the maximum number of allowed
   bosonic states and a tolerance for discarding imaginary parts of the coefficients.
 
-<h4>Construct vibrational Hamiltonians 🫨</h4>
-
-* Added submodule for calculating vibrational Hamiltonians
-  * Implemented helper functions for geometry optimization, harmonic analysis,
-    and normal-mode localization.
-    [(#6453)](https://github.com/PennyLaneAI/pennylane/pull/6453)
-    [(#6666)](https://github.com/PennyLaneAI/pennylane/pull/6666)
-  * Implemented wrapper function for vibrational Hamiltonian calculation and dataclass
-    for storing the data.
-    [(#6652)](https://github.com/PennyLaneAI/pennylane/pull/6652)
-  * Implemented functions for generating the vibrational Hamiltonian in VSCF basis
-    [(#6688)](https://github.com/PennyLaneAI/pennylane/pull/6688)
-
-* Added support to build a vibrational Hamiltonian in Taylor form.
+<h4>Construct vibrational Hamiltonians 🔨</h4>
+
+* The new `pennylane.qchem.vibrational` submodule contains several features to help with the
+construction of vibrational Hamiltonians. This includes:
+  * The `VibrationalPES` class to store potential energy surface information. 
```

Comment:
```suggestion
* The new :mod:`qml.qchem.vibrational <pennylane.qchem.vibrational>` submodule contains several features to help with the
construction of vibrational Hamiltonians. This includes:
  * The :class:`~.VibrationalPES` class to store potential energy surface information:
```

Reply:
Wait... is there a `vibrational` module? 🤔

Reply:
It doesn't appear in the docs :cry:

Reply:
Ruh roh, should it? 🐶

Reply:
The `__init__` file is very sparse for it, I don't think it's intended to show.
I remember Austin saying something about a pending ticket to update the documentation in Q1. Probably related to this.

Reply:
```suggestion
* The :mod:`~.qchem` module now contains several features to help with the
construction of vibrational Hamiltonians. This includes:
  * The `VibrationalPES` class to store potential energy surface information. 
```